.babelrc

```diff
@@ -0,0 +1,19 @@
+{
+  "plugins": [
+    ["transform-runtime", {
+      "polyfill": false,
+      "regenerator": true
+    }]
+  ],
+  "presets": [["env", {
+    "targets": {
+      "browsers": ["last 2 versions", "safari >= 7", "ie 9"],
```

Comment:
It is better to replace this with single `package.json`. property:

```json
  "browserslist": ["last 2 versions", "safari >= 7", "ie 9"],
```

See https://github.com/ai/browserslist

In fact, it is a recommended by developers method.